.github/patches/release/v1.25.x/opentelemetry-java-instrumentation.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/dependencyManagement/build.gradle.kts b/dependencyManagement/build.gradle.kts
+index 954167eab5..ca892b1499 100644
+--- a/dependencyManagement/build.gradle.kts
++++ b/dependencyManagement/build.gradle.kts
+@@ -12,7 +12,7 @@ val dependencyVersions = hashMapOf<String, String>()
+ rootProject.extra["versions"] = dependencyVersions
+ 
+ // this line is managed by .github/scripts/update-sdk-version.sh
+-val otelSdkVersion = "1.25.0"
++val otelSdkVersion = "1.25.0-adot1"
+ 
+ // Need both BOM and groovy jars
+ val groovyVersion = "4.0.11"
+@@ -29,7 +29,7 @@ val groovyVersion = "4.0.11"
+ // configurations.testRuntimeClasspath.resolutionStrategy.force "com.google.guava:guava:19.0"
+ 
+ val DEPENDENCY_BOMS = listOf(
+-  "com.fasterxml.jackson:jackson-bom:2.14.2",
++  "com.fasterxml.jackson:jackson-bom:2.15.0",
```

Comment:
Should we be going to 2.15.1?

Reply:
we can't https://github.com/open-telemetry/opentelemetry-java/pull/5459

Reply:
dang jackson! rude.